examples/textinput/textinput/app.py

```diff
@@ -29,11 +30,11 @@ def do_extract_values(self, widget, **kwargs):
             self.password_input.value,
         )
 
-        number = self.number_input.value
-        if number:
+        number = self.number_input.value + self.right_aligned_number_input.value
+        if number > 0:
             self.number_label.text = f"Double the number is: {number * 2}"
         else:
-            self.number_label.text = "You didn't enter a number"
+            self.number_label.text = "Pick some numbers that add up to more than zero"
```

Comment:
The old logic exited because number can be blank, which means addition and doubling won't work.

This new example will crash if *either* value doesn't exist. It would also make sense to report the *sum* of the two values, rather than double *one* of them.

Reply:
I've changed the logic completely  to address this:
- label shows values for each box and sum
- added basic error handling for null values by displaying a warning to add a number to input box